newrelic-nri-kube-events.yaml

```diff
@@ -35,7 +35,69 @@ update:
     tag-filter: v
 
 test:
+  environment:
+    contents:
+      packages:
+        - kubectl
```

Comment:
nit: we dont need to define kubectl

Reply:
We need it to set up tokens and service accounts. Did you mean that it's already included in test environment?